.ghci

```diff
@@ -6,6 +6,6 @@
 
 :def loadPresentation \p -> return $ "Presentation {..} <- loadPresentation \"" ++ p ++ "\""
 
-:def loadTestPresentation const . return $ "Presentation {..} <- loadPresentation \"/Users/Frantisek/haskell/howerpoint/test/resources/howerpoint.txt\""
+:def loadTestPresentation const . return $ "Presentation {..} <- loadPresentation \"test/resources/howerpoint.txt\""
```

Comment:
specifying relative-path works, and has the additional benefit that it works on other peoples' computers